specification/src/main/asciidoc/platform/CompatibilityMigration.adoc

```diff
@@ -35,30 +35,30 @@ version of the platform.
 Migration is the act of converting an
 application to use new facilities introduced in this release of the
 platform. Given the strong level of compatibility in this release of the
-Java EE platform, migration is largely an optional exercise. Still, an
+Jakarta EE platform, migration is largely an optional exercise. Still, an
 application may be improved (better performance, simpler to develop,
 more flexible, etc.) by converting it to use newer facilities of the
-Java EE platform.
+Jakarta EE platform.
 
-=== Java Persistence
+=== Jakarta Persistence
 
-Java Persistence provides a much richer set of
+Jakarta Persistence provides a much richer set of
 modeling capabilities and object/relational mapping capabilities than
 EJB CMP entity beans and is significantly easier to use.
 
 Support for EJB CMP and BMP entity beans has
 been made optional with the Java EE 7 release. Support for EJB CMP 1.1
 entity beans has been deprecated since Java EE 5. Applications are
 strongly encouraged to migrate applications using EJB entity beans to
-the Java Persistence API.
+Jakarta Persistence.
 
-=== JAX-WS
+=== Jakarta XML Web Services
 
-JAX-WS, along with JAXB and the Metadata for
+Jakarta XML Web Services, along with Jakarta XML Binding and the Metadata for
```

Comment:
Neither JAX-WS nor JAXB are part of Jakarta EE 8.
This section should continue to reference the original spec names.